doc/source/releases/20.05_announce.rst

```diff
@@ -0,0 +1,49 @@
+
+===========================================================
+May 2020 Galaxy Release (v 20.05)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+**Workflow Editor and Workflow Run Form in Vue.js**
+  @jmchilton, @guerler, can you fill this in ?
+  Feature description.
+
+**Data Tables can now be backed by refgenie**
+  @blankenberg, can you write a line about this
+
+**Tool Shed is now Python3 ready**
+  @mvdbeek will fill this
+
+**Many new Interactive Tools**
+  @astrovsky01 can you write a bit abbout this ?
```

Comment:
Galaxy interactive tools allow a greater depth of analysis via access to GUI-based tools inside an instance. 10 new interactive tools join the standard base toolset on galaxy download. Several tools previously only available on the usegalaxy.eu server, are now able to be run on local instances via configurations in the job_conf.xml and galaxy.yml files, details of which can be found in the documentation under "Galaxy InteractiveTools". Improvements to the Higlass interactive tool also now allow for a more in-depth visualization of multiple datasets in various formats simultaneously (`Pull Request 9559`_).